### PR TITLE
override 1config home by setting ONECONFIG_HOME in the environment

### PR DESCRIPTION
**Pull request by sathyavijayan**

### CODE REVIEW

File: 1config-core/src/com/brunobonacci/oneconfig/util.clj [hunk 1]
```diff
@@ -235,15 +235,27 @@
...
+  [path]
+  (when-let [dir (some-> path io/file)]
+    (and (.exists ^File dir) (.isDirectory ^File dir))))
+
 
 (defn home-1config
-  "returns ~/.1config/"
+  "if ONECONFIG_HOME is set in the environment, it returns
+  <ONECONFIG_HOME>/.1config, otherwise it returns ~/.1config"
   []
```
Comment (BrunoBonacci) [inline 1config-core/src/com/brunobonacci/oneconfig/util.clj]: I think that the behaviour of this function should be more like if `ONECONFIG_HOME` then it is returned,
if it isn't defined then it does what it was doing before like: `$HOME/`+`.1config`
so it makes it clear that when you define your `export ONECONFIG_HOME=/foo`, `/foo` is going to be used as 1Config home and if not defined by default is : `$HOME/`+`.1config`

Comment (BrunoBonacci) [inline 1config-core/src/com/brunobonacci/oneconfig/util.clj]: I think that the behaviour of this function should be more like if `ONECONFIG_HOME` then it is returned,
if it isn't defined then it does what it was doing before like: `$HOME/`+`.1config`
so it makes it clear that when you define your `export ONECONFIG_HOME=/foo`, `/foo` is going to be used as 1Config home and if not defined by default is : `$HOME/`+`.1config`

File: ver/1config.version [hunk 2]
```diff
@@ -1 +1 @@
-0.15.0
+0.15.1
```
[the remark this answers is not in the document]
Comment (sathyavijayan) [reply ver/1config.version]: Done.

File: 1config-core/src/com/brunobonacci/oneconfig/util.clj (hunk 1 above)
Comment (sathyavijayan) [reply 1config-core/src/com/brunobonacci/oneconfig/util.clj]: Agreed, API is much cleaner that way. I have done the changes.

**Pull request merged**